components/tor/onion_location_navigation_throttle.cc

```diff
@@ -88,14 +83,8 @@ OnionLocationNavigationThrottle::WillProcessResponse() {
             navigation_handle()->GetWebContents())) {
       return content::NavigationThrottle::PROCEED;
     }
-    // If user prefers opening it automatically
-    if (ShouldAutoRedirect()) {
-      delegate_->OpenInTorWindow(navigation_handle()->GetWebContents(), url);
```

Comment:
It seems that we no longer need this delegate, it can be completely deleted.

Reply:
Good catch, removed.